schema/password.go

```diff
@@ -58,9 +58,9 @@ func (v Password) Validate(value interface{}) (interface{}, error) {
 // VerifyPassword compare a field of an item payload containing a hashed
 // password with a clear text password and return true if they match.
 func VerifyPassword(hash interface{}, password []byte) bool {
-	h, ok := hash.([]byte)
+	h, ok := hash.(string)
 	if !ok {
 		return false
 	}
-	return bcrypt.CompareHashAndPassword(h, password) == nil
```

Comment:
I want to suggest this instead:
```go
var h []byte
switch hashT := hash.(type) {
case []byte:
    b = hashT
case string:
    b = []byte(hashT)
default:
    return false
}
return bcrypt.CompareHashAndPassword(h, password) == nil
```

Then we handle both a `[]byte` and a `string` DB value, which both probably can be the case, depending on schema and/or backend.